plugins/email_service/app/controllers/email_service/settings_controller.rb

```diff
@@ -1,28 +1,30 @@
 module EmailService
   class SettingsController < ::EmailService::ApplicationController
-    # before_action :restrict_access
+
+    before_action :check_ec2_creds_cronus_status
+    before_action :check_verified_identity
 
     authorization_context 'email_service'
     authorization_required
-    
+
     def index
       @nebula_details = nebula_details
       @nebula_status = nebula_status
-      puts @nebula_status
-      puts @nebula_details
+      # Rails.logger.debug @nebula_status
+      # Rails.logger.debug @nebula_details
       @nebula_endpoint = "https://nebula.#{current_region}.cloud.sap"
```

Comment:
I still see the nebula endpoint as a string in the code. Please fetch the endpoint fron the catalog

Reply:
removed.